### ISSUE TITLE
WCAG section 508 issues

### ISSUE THREAD

**Issue by hzhao392** [episphere/connectApp#50]
Hi @naiyume here are some of the WAVE tool findings, can you add the missing form labels? Let me know if you need help running the wave tool

![image](https://user-images.githubusercontent.com/57492029/114081227-4b330900-987a-11eb-87c5-43673c6e1364.png)

**Comment by anthonypetersen** [comment]
No recent activity
